src/Halogen/Aff/Driver/Eval.purs

```diff
@@ -38,10 +38,7 @@ import Halogen.Subscription as HS
 import Unsafe.Reference (unsafeRefEq)
 
 type Renderer r
-  = forall s f act ps i o
-   . Ref LifecycleHandlers
-  -> Ref (DriverState r s f act ps i o)
-  -> Effect Unit
+  = forall s f act ps i o. Ref LifecycleHandlers -> Ref (DriverState r s f act ps i o) -> Effect Unit
```

Comment:
It would be nice to still have this break over lines, but it's not awful to be all on one line either.